pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -1303,6 +1312,10 @@ func buildDefaultCluster(push *model.PushContext, name string, discoveryType api
 	}
 
 	if discoveryType == apiv2.Cluster_STATIC || discoveryType == apiv2.Cluster_STRICT_DNS {
+		if len(localityLbEndpoints) == 0 {
+			log.Errorf("STRICT_DNS/STATIC Cluster:%s, does not have endpoints, skipped", name)
```

Comment:
Remove the error message please as this is going to get very noisy. Instead add it as a metric (look around pilot code for how we do metrics for these kinds of things).

Reply:
Done